roles/openshift_control_plane/tasks/set_loopback_context.yml

```diff
@@ -16,6 +16,17 @@
   - loopback_context_string not in l_loopback_config.stdout
   register: set_loopback_cluster
 
+- command: >
+    {{ openshift_client_binary }} config set-credentials
+    --client-certificate=/etc/origin/master/openshift-master.crt
+    --client-key=/etc/origin/master/openshift-master.key
+    --embed-certs=true
+    {{ openshift.master.loopback_user }}
+    --config={{ openshift_master_loopback_config }}
+  when:
+  - set_loopback_cluster | changed
```

Comment:
I think the filter syntax "| changed" is deprecated, should use "is changed" I believe.